driver/sccb.c

```diff
@@ -20,6 +24,25 @@
 static const char* TAG = "sccb";
 #endif
 
+// Possible SCCB ports
+static const uint8_t SCCBPort[]=
+{
+#if CONFIG_OV7670_SUPPORT || CONFIG_OV7725_SUPPORT
+0x42,
+#endif
+#if CONFIG_NT99141_SUPPORT
+0x54,
+#endif
+#if CONFIG_OV2640_SUPPORT
+0x60,
+#endif
+#if CONFIG_OV3360_SUPPORT || CONFIG_OV5640_SUPPORT
+0x78,
+#endif
```

Comment:
please do not pre-shift the addresses :) I know that they are not user accessible, but having them with values that are known to people makes a bit more sense